src/lib/db/DatabaseManager.ts

```diff
@@ -1,22 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import { addRxPlugin } from "rxdb";
 import { RxDBDevModePlugin } from "rxdb/plugins/dev-mode";
 import { getRxStorageDexie } from "rxdb/plugins/storage-dexie";
 import { createRxDatabase, RxDatabase } from "rxdb";
 import { v4 as uuidv4 } from "uuid";
-
 import toolkitItemSchema from "./schemas/toolkitItemSchema.json";
 import moodRecordSchema from "./schemas/moodRecordSchema.json";
 import categoriesSchema from "./schemas/categoriesSchema.json";
 import needsCategoriesSchema from "./schemas/categoriesSchema.json";
 import needsSchema from "./schemas/categoriesSchema.json";
 import nextActionsSchema from "./schemas/categoriesSchema.json";
 
-import { categories, toolkit } from "./seed/toolkit";
-import { needsCategories, needs, nextActions } from "./seed/needs";
+// import { categories, toolkit } from "./seed/toolkit";
+// import { needsCategories, needs, nextActions } from "./seed/needs";
 
```

Comment:
uncomment these so that the seeding can work
```suggestion
import { categories, toolkit } from "./seed/toolkit";
import { needsCategories, needs, nextActions } from "./seed/needs";

```